workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer/README.md

```diff
@@ -0,0 +1,176 @@
+
+```
+/* highlight-add-next-line */
+import {
+  isMultiCIAvailable,
+  EntityMultiCIPipelinesContent,
+} from '@backstage-community/plugin-multi-source-security-viewer';
+
+const cicdContent = (
+  <EntitySwitchj
+    {/* ... */}
+    {/* highlight-add-start */}
+    <EntitySwitch.Case if={isMultiCIAvailable}>
+      <EntityMultiCIPipelinesContent />
+    </EntitySwitch.Case>
+    {/* highlight-add-end */}
+  </EntitySwitch>
+);
+```
+
+## For users
+
+### Using the plugin in Backstage
+
+#### Prerequisites
+
+1. Your backstage application is installed and running.
+2. You have installed the Multi Source Security Viewer plugin.
+3. Ensure your CI annotations are set:
+
+- [Jenkins](../../../jenkins/plugins/jenkins/README.md)
+- [Github Actions]([../../../github-actions/plugins/github-actions/README.md)
+- [Gitlab CI](https://github.com/immobiliare/backstage-plugin-gitlab?tab=readme-ov-file#annotations)
+
+### Procedure
+
+1. Open your Backstage application and select a component from the **Catalog** page.
+2. Go to the **CI/CD** tab.
+
+The **CI/CD** tab displays the list of Pipeline Runs from your CI's based on the component annotations.
+
+![ci-cd-tab-mssv](./docs/images/mssv-multi-ci.png)
+
+The table provides a visual representation of identified vulnerabilities in the OCI image produced by the pipelinerun.
+The Author of the pipeline scanner task would provide the CVE summary data using the below format that the UI can interpret.
+
+### Actions buttons
+
+#### SBOM
+
+Link to SBOM action will be enabled if there is a SBOM task in the pipeline or a external link if provided by the eye catcher.
+The sbom is captured from the `show-sbom-rhdh` [step](https://github.com/redhat-appstudio/tssc-dev-multi-ci/blob/main/rhtap/show-sbom-rhdh.sh).
+
+#### Logs
+
+Pipeline steps and their logs are visible if steps have been identified.
+
+![ci-cd-tab-mssv-logs](./docs/images/logs-dialog.png)
+
+#### Output
+
+Output action will be enabled when the pipeline emits some results and/or contains its eye catchers.
+
+This action opens a modal where it will render the reports for Enterprise contract and Advanced cluster security. The report data should be exposed via logs through EYE_CATCHERS.
+
+Supported EYE_CATCHERS:
+
+- ACS_IMAGE_SCAN_EYECATCHER_BEGIN
+- ACS_IMAGE_SCAN_EYECATCHER_END
+- ACS_IMAGE_CHECK_EYECATCHER_BEGIN
+- ACS_IMAGE_CHECK_EYECATCHER_END
+- ACS_DEPLOY_EYECATCHER_BEGIN
+- ACS_DEPLOY_EYECATCHER_END
+- EC_EYECATCHER_BEGIN
+- EC_EYECATCHER_END
+
+![ci-cd-tab-mssv-output](./docs/images/mssv-output.png)
+
+### Format:
+
+The data is extracted from logs. The steps are identified with `Step : <ANY>`.
+eg: `Step: cosign-sign-attest`.
+
+Results are extracted between EYE_CATCHERS as json.
+eg: from `ACS_IMAGE_SCAN_EYECATCHER_BEGIN` to `ACS_IMAGE_SCAN_EYECATCHER_END`
+
+Resuls output:
+
+`{
+  results: [
+    {
+      summary: {
+        CRITICAL: 0,
+        HIGH: 1,
+        LOW: 1,
+        MEDIUM: 0,
+        TOTAL: 2,
+      }
+    }
+  ]
```

Comment:
I think, we should add more documentation for acs image_scan, image_check, deployment_check and for the enterprise_contract types. 

All the above scan outputs has some differences

[AcsImageScan](https://github.com/redhat-developer/aonic-ui/blob/main/packages/pipelines/src/components/Output/types.ts#L126C18-L131C2)

[AcsImageCheck](https://github.com/redhat-developer/aonic-ui/blob/main/packages/pipelines/src/components/Output/types.ts#L126C18-L131C2)

[EnterpriseContractResult](https://github.com/redhat-developer/aonic-ui/blob/main/packages/pipelines/src/components/Output/types.ts#L126C18-L131C2)

what do you think about having some links to the mock fixtures as an examples?